README.md

```diff
@@ -1,5 +1,5 @@
 # Theos Wiki
-This repository manages the contents of [Theos’s GitHub Wiki](https://github.com/theos/theos/wiki). You can submit pull requests to it here, and it will be deployed automatically via GitHub Actions.
+This repository manages the contents of [Theos’s GitHub Wiki](https://github.com/theos/theos/wiki). You can submit pull requests to it here and it will be deployed automatically via GitHub Actions.
```

Comment:
The comma which was there before was perfectly fine

Reply:
It's fine either way. A comma though will be used when there are at least three different ideas in a sentence.

Reply:
I’d prefer to stick to Oxford commas for improved clarity.

Reply:
Alright. Reverted my change.